src/Wordpress/CustomPostType/CustomPostType.php

```diff
@@ -241,34 +240,23 @@ public function initListView() {
 			$this,
 			'setSortableColumns'
 		) );
+
 		if ( isset( $this->listColumns ) ) {
-			add_action( 'pre_get_posts', function ( $query ) {
-				if ( ! is_admin() ) {
-					return;
-				}
+			add_action( 'pre_get_posts', array( $this, 'setCustomColumnSortOrder' ) );
+		}
 
-				//TODO: This does correctly sort the items by meta value, since WP 6.3 the meta value is not passed to the query anymore. Maybe the filter needs to be changed?
-				$orderby = $query->get( 'orderby' );
-				//Prior to WP 6.3, this was not an associative array (see #1309) but a string
-				if (is_array($orderby)) {
-					$orderKeys = array_keys( $orderby );
+		// add ability to use WP_QUERY orderby for post_status
```

Comment:
@hansmorb wenn ich es richtig verstehe macht dieser Teil ja explizit Nutzen von der Ausnahme über den Filter `posts_orderby` für Felder die nicht im Post sondern in einer anderen Tabelle (in unserem Fall Meta) sind. Aber standardmäßig kann ich doch `title` als `orderby` für die query-args mitgeben.

Wieso sagst du geht das nicht?

https://developer.wordpress.org/reference/classes/wp_query/#order-orderby-parameters

Reply:
Hmm, ich habe diesen Codeschnipsel einfach von irgendwoher kopiert. So ganz sicher bin ich mir da auch nicht. Bau gerne was um wenn das eleganter geht.